src/libs/IntlPolyfill/index.ios.ts

```diff
@@ -7,12 +7,21 @@ import type IntlPolyfill from './types';
  * Polyfill the Intl API, always performed for native devices.
  */
 const intlPolyfill: IntlPolyfill = () => {
-    // Native devices require extra polyfills
-    require('@formatjs/intl-getcanonicallocales/polyfill');
+    // Native devices require extra polyfills which are
+    // not yet implemented in hermes.
+    // see support: https://hermesengine.dev/docs/intl/
+
     require('@formatjs/intl-locale/polyfill');
+
+    // Required to polyfill NumberFormat on iOS
+    // see: https://github.com/facebook/hermes/issues/1172#issuecomment-1776156538
     require('@formatjs/intl-pluralrules/polyfill');
```

Comment:
Isn't this needed for android too? It's listed as not supported https://hermesengine.dev/docs/intl/#not-yet-supported

Reply:
@s77rt  that's correct. But `PluralRules` is required only if we are polyfilling `NumberFormat`, see [here](https://formatjs.io/docs/polyfills/intl-numberformat) and since we aren't polyfilling `NumberFormat` on android, we don't need to polyfill `PluralRules` there.